userspace/libsinsp/test/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@
 include_directories("${GTEST_INCLUDE_DIR}")
 include_directories("${JSONCPP_INCLUDE}")
 include_directories("${TBB_INCLUDE_DIR}")
+include_directories("${CURL_INCLUDE_DIR}")
```

Comment:
Was this forgotten before or is it a new (pretty well hidden) dependency?

Reply:
This was broken when I merged... I'll check if it is still needed.

Reply:
Looks like this is still needed but I moved it to a different PR